src/presence_analyzer/tests.py

```diff
@@ -46,7 +46,28 @@ def test_mainpage(self):
         """
         resp = self.client.get('/')
         self.assertEqual(resp.status_code, 302)
-        assert resp.headers['Location'].endswith('/presence_weekday.html')
+        assert resp.headers['Location'].endswith('/presence_weekday')
+
+    def test_mean_time_weekday(self):
```

Comment:
Three separate tests wouldn't be needed if you figure out solution to make it a single view. See bottom comment.